src/main/features/response/routes/response-type.ts

```diff
@@ -35,10 +35,20 @@ export default express.Router()
         user.responseDraft.document.response = form.model
         await DraftService.save(user.responseDraft, user.bearerToken)
 
-        if (ResponseType.OWE_NONE === form.model.type) {
-          res.redirect(Paths.defenceOptionsPage.evaluateUri({ externalId: externalId }))
-        } else {
-          res.redirect(Paths.taskListPage.evaluateUri({ externalId: externalId }))
+        const responseType = user.responseDraft.document.response.type
+
+        switch (responseType) {
+          case ResponseType.OWE_NONE:
+            res.redirect(Paths.defenceRejectAllOfClaimPage.evaluateUri({ externalId: externalId }))
+            break
+          case ResponseType.OWE_SOME_PAID_NONE:
+            res.redirect(Paths.defenceRejectPartOfClaimPage.evaluateUri({ externalId: user.claim.externalId }))
+            break
+          case ResponseType.OWE_ALL_PAID_NONE:
+            res.redirect(Paths.taskListPage.evaluateUri({ externalId: user.claim.externalId }))
+            break
+          default:
+            next(new Error('Unknown response type: ' + responseType))
```

Comment:
Backtics can be used here to interpolate the string.

Reply:
Done